package.json

```diff
@@ -16,17 +16,14 @@
       "jquery": true
     },
     "globals": {
-      // 1. Table names
```

Comment:
```bash
npm ERR! Linux 4.2.0-16-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "test"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! file /media/mariam/Files/Work/question_tool/package.json
npm ERR! code EJSONPARSE

npm ERR! Failed to parse json
npm ERR! Unexpected token '/' at 19:7
npm ERR!       // hello world
npm ERR!       ^
npm ERR! File: /media/mariam/Files/Work/question_tool/package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
```
Apparently, [comments in JSON are invalid](http://stackoverflow.com/questions/244777/can-comments-be-used-in-json).